SQL/0000-00-00-schema.sql

```diff
@@ -2014,3 +2018,40 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- ********************************
+-- Consent tables
+-- ********************************
+
+CREATE TABLE `consent` (
+  `ConsentID` int(2) NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  CONSTRAINT `PK_consent` PRIMARY KEY (`ConsentID`),
+  CONSTRAINT `UK_consent_Name` UNIQUE KEY `Name` (`Name`),
+  CONSTRAINT `UK_consent_Label` UNIQUE KEY `Label` (`Label`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `candidate_consent_rel` (
+  `CandidateID` int(6) NOT NULL,
+  `ConsentID` int(2) NOT NULL,
+  `Status` enum('yes','no') DEFAULT NULL,
+  `DateGiven` date DEFAULT NULL,
```

Comment:
For a yes/no attributes, could use bit(1) with "0" and "1" as the sole possible values. I know that @driusan  doen't like boolean attributes.

Reply:
I think @driusan doesn't like the Boolean simply because analysing the data when you query it can become error prones since 1's and 0's are interpreted by PHP differently...

Reply:
@ridz1208  Boolean and BIT(1) is not the same. BOOLEAN are treated as TINYINT(1).
@driusan Could you explain why you do not like BOOLEAN.  Would BIT(1) be Ok for pass/fail, yes/no choices?

Reply:
*I* like boolean attributes, the combination of MySQL turning booleans into tinyint(1), POST requests turning everything into a string, and PHP having weird truthiness and printing characteristics for "0" which doesn't like boolean and has introduced multiple bugs in the past.

Reply:
(and I don't see how BIT(1) would be different than TINYINT(1) in that regard...)

Reply:
It take 2 possible value string "0"  or string "1"